src/policy/CMSRucioPolicy/permission.py

```diff
@@ -232,17 +233,17 @@ def _get_rule_size(rules):
 
     # Check if the account is banned
     if has_account_attribute(account, 'auto_approve_banned', session=session):
-        return False
+        return PermissionResult(False, "The account is banned.")
 
     # Check if the rule is locked
     if kwargs['locked']:
-        return False
+        return PermissionResult(False, "The rule is locked.")
 
     if kwargs['lifetime'] is None:
-        return False
+        return PermissionResult(False, "lifetime argument is None.")
 
     if kwargs['lifetime'] > rule_lifetime_threshold:
-        return False
+        return PermissionResult(False, "The rule lifetime exceeds the treshold.")
```

Comment:
Since these messages are for users, robots do not use the "user autoapproce" activity (I hope, haha).
We should point them to the docs that explain and mention the thresholds (or you can print the limit here too if you want).

Reply:
Same this applies to other messages as well.